source/common/common/posix/thread_impl.cc

```diff
@@ -97,7 +97,7 @@ class ThreadImplPosix : public Thread {
   // Attempts to get the name from the operating system, returning true and
   // updating 'name' if successful. Note that during normal operation this
   // may fail, if the thread exits prior to the system call.
-  bool getNameFromOS(std::string& name) {
```

Comment:
this is an output param

Reply:
What is an output parameter? Can you please explain.